README.md

```diff
@@ -83,15 +83,15 @@ To get started developing on Record Store, run `bin/setup`. This will create a d
 
 ### Adding new Providers
 
-To add a new Provider, create a class inherriting `Provider` in [`lib/recordstore/provider/`](lib/recordstore/provider/). The [DynECT provider](lib/recordstore/provider/dnsimple.rb) is good to use as a reference implementation.
+To add a new Provider, create a class inherriting `Provider` in [`lib/record_store/provider/`](lib/record_store/provider/). The [DynECT provider](lib/record_store/provider/dnsimple.rb) is good to use as a reference implementation.
```

Comment:
Good catch, it should be a different file path. DynECT was the original implementation and doesn't have as much data shaping.